app/models/form/question.rb

```diff
@@ -41,6 +53,16 @@ def enabled?(case_log)
     conditional_on.map { |condition| evaluate_condition(condition, case_log) }.all?
   end
 
+  def hidden_in_check_answers?
+    hidden_in_check_answers
+  end
+
+  def has_inferred_check_answers_value?(case_log)
+    return inferred_check_answers_value["condition"].values[0] == case_log[inferred_check_answers_value["condition"].keys[0]] if inferred_check_answers_value.present?
```

Comment:
Is this definitely always a hash with only a single key/value?

Reply:
It is right now, cause there's only one such occurrence

Reply:
I think that's fine then but worth documenting that assumption in the definition schema doc in the readme?